### PR TITLE
Added copyright info to scripture

### PR DESCRIPTION
**Pull request by redreceipt**
Fixes [#309]

Researched copyright requirements defined by [API.Bible](http://el.bibles.org/pages/legal/#terms-api) and added the call in our API and render it out in our components using a custom `LegalText` component.

API.Bible also has a system they use for analytics called the [Fair Use Management System](http://el.bibles.org/pages/api/documentation/fums). They give an embeddable tag to use in a static page but our conclusion is that it's not worth implementing in our server unless they can give us a better node-y solution. Waiting to hear back from customer support.

For Future implementation, with the ability to choose a translation, we will want to make it prettier (see the attached KJV example below). Currently, with WEB, we don't need to display anything. I've made another issue (#325) to track this feature.

screen shot 2018-10-11 at 11 01 33 am
screen shot 2018-10-11 at 11 27 43 am
screen shot 2018-10-11 at 11 02 03 am

## Creator

- [x] Build relevant tests if any apply
- [ ] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [ ] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

**Comment by vinnyjth** [comment]
Hey @redreceipt :D 

A few tips so you can get this over the finish line :) 

- As soon as you want people to review your PR, feel free to add the "ready for review" flag. That way people know that you're ready for us to approve your PR :) 
- This isn't a tip, I'm just really proud of you including three screenshots. That's very helpful for the reviewers. 
- It looks like your tests are failing. It's probably because the snapshots don't match anymore. In thise case, that's good, because we added new data. Run `yarn jest -u` inside your `apolloschurchapp` folder and then commit the changes :) 

Besides that, your PR looks great so far.

File: packages/apollos-church-api/src/data/bible/resolver.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ export default {
       dataSources.Scripture.getScripture(query),
   },
   Scripture: {
-    reference: ({ data: { reference } = {} }) => reference,
     html: ({ data: { passages } = {} }) => get(passages, '[0].content'),
+    reference: ({ data: { passages } = {} }) => get(passages, '[0].reference'),
```
Comment (vinnyjth) [inline packages/apollos-church-api/src/data/bible/resolver.js]: These lines are't currently "covered". That means that our test suite never executes them. You can fix this by adding them to the query in

```
apollos-church-app/apollos-church-api/src/data/bible/_tests_/resolver.tests.js
```

and running `yarn jest -u` in the `apollos-church-app` folder.

Comment (redreceipt) [reply packages/apollos-church-api/src/data/bible/resolver.js]: added

Comment (redreceipt) [reply packages/apollos-church-api/src/data/bible/resolver.js]: the question is @vinnyjth how did you figure that out and how can I in the future :)

Comment (vinnyjth) [reply packages/apollos-church-api/src/data/bible/resolver.js]: Once the tests are run, there's a link to CodeCov, which is a tool that determines code coverage. We aim to get 100% coverage over the app, the API and Client are measured separately.  

There's also a comment earlier in the PR, autoposted, that shows an overview of the coverage. 
screenshot 2018-10-11 14 44 45

**Pull request merged**